src/app/App.svelte

```diff
@@ -1,11 +1,23 @@
 <script>
+  import { init, dictionary } from "svelte-i18n";
   import { Router, Route } from "svelte-routing";
   import LandingPage from "./pages/LandingPage.svelte";
   import Languages from "./pages/Languages.svelte";
   import Privacy from "./pages/Privacy.svelte";
   import Team from "./pages/Team.svelte";
   import Help from "./pages/Help.svelte";
 
+  import en from "./lang/en";
+  import ko from "./lang/ko";
+
+  const [initialLocale] = window.navigator.languages;
```

Comment:
This is a workaround for `getClientLocale` available in `svelte-i18n` package as I couldn't import it somehow, might be related to [this thread](https://github.com/kaisermann/svelte-i18n/commit/31b556bc3f77bc5b581541976a82f898a398c01a#diff-3c1e03e5a3ca9dc9415b55abb4e92f4d1d6975660903414c68d59b658143f642).

Reply:
Weird. I was gonna say, it would be nice to use `getClientLocale()` but it appears to [not really do much more than this code](5cf5c938b2ae4641a8271e4c99ceea27c741f76a) anyway 🤔 

Acceptable!